flink-core/src/main/java/org/apache/flink/configuration/ConfigConstants.java

```diff
@@ -1333,6 +1333,11 @@
 		key("jobmanager.web.address")
 			.noDefaultValue();
 
+	/** Web response header of Access-Control-Allow-Origin's key and default value. */
```

Comment:
Please replace this with "The config parameter defining the Access-Control-Allow-Origin header for all responses from the web-frontend".